packages/inventory/src/Stocktake/api/hooks.ts

```diff
@@ -107,14 +110,40 @@ export const useStocktakeRows = (isGrouped = true) => {
   });
   const { data: lines } = useStocktakeLines();
   const { data: items } = useStocktakeItems();
+  const columns = useStocktakeColumns({ onChangeSortBy, sortBy });
+
+  const sortedItems = useMemo(() => {
+    const currentColumn = columns.find(({ key }) => key === sortBy.key);
+    if (!currentColumn?.getSortValue) return items;
+    const sorter = getColumnSorter(
+      currentColumn?.getSortValue,
+      !!sortBy.isDesc
+    );
+    return items?.sort(sorter);
+  }, [items, sortBy.key, sortBy.isDesc]);
+
+  const sortedLines = useMemo(() => {
```

Comment:
wasn't sure why the `sortedLines` aren't used below ( line 135 ). the lines are sorted again down there.
the sortedLines aren't used yet, so I couldn't guess intention from usage 😄

Reply:
whoops 🙄